packages/muze/src/canvas/layout-maker.js

```diff
@@ -172,7 +174,16 @@ const componentNames = {
     2: LEGEND,
     3: VERTICAL_SCROLL_BAR,
     4: HORIZONTAL_SCROLL_BAR,
-    5: GRID
+    5: GRID,
+    6: MESSAGE
+};
+
+const attachListeners = (componentWrappers) => {
+    componentWrappers.forEach((componentWrapper) => {
+        if (componentWrapper) {
```

Comment:
How can `componentWrapper` be undefined here?

Reply:
It's possible that componentWrapper is not present or is null. In such a case, the wrapper will not be created.